_data/plugins/katello.yml

```diff
@@ -0,0 +1,31 @@
+versions:
+  - name: "3.13"
+    display: "3.13"
+  - name: "3.12"
+    display: "3.12"
+  - name: "3.11"
+    display: "3.11"
+  - name: "3.10"
+    display: "3.10"
+  - name: 3.9
```

Comment:
This would require updating the 'version' value in all the older docs markdown files so the comparison can be made correctly. For now I'd rather leave it and figure out the plan for those older docs (such as archiving them).

Reply:
Ok, fair enough.